Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,48 @@
-﻿using System;
-
-namespace BootCamp.Chapter
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string name, surname, ageInput, weightInput, heightInput;

```

Comment:
It's recommended to just declare a variable right before you use it and not all in one place. Like that, you will have the variable at the place where you need to see it and not scroll up and down.